ts/session/messages/outgoing/index.ts

```diff
@@ -1,11 +1,44 @@
-import { OutgoingMessage } from './OutgoingMessage';
-import { OutgoingContentMessage } from './OutgoingContentMessage';
-import { OutgoingDataMessage } from './OutgoingDataMessage';
+import { Message } from './Message';
+import { ContentMessage } from './content/ContentMessage';
```

Comment:
it might make more sense to export specific messages based on the folder and then import them all here in one go.
For example, you would create `index.ts` inside `./content` you would have
```ts
import * as DataMessages from './data'
import * as SyncMessages from './sync'
import { TypingMessage } from './TypingMessage'

export {
    ...DataMessages,
    ...SyncMessages,
    TypingMessage
}
```

and in this you would have
```
import * as ContentMessages from './content'
export {
    ...ContentMessages
}
```

**I don't know if this is standard practice though so it might be better to look into it more**

Reply:
Fixed (but the spread operator is not valid for exports/)